docs/config/README.md

```diff
@@ -1328,6 +1328,44 @@ By default the module will be shown if any of the following conditions are met:
 format = "via [🏎💨 $version](bold cyan) "
 ```
 
+## Haskell
+
+The `haskell` module shows the currently installed version of Haskell.
+By default the module will be shown if any of the following conditions are met:
+
+- The current directory has a file with the extension of `.hs`, `.hsc` or `.hs-boot`.
```

Comment:
I believe the criteria should be if there is a `stack.yaml` or  `foo.cabal` present at the project root.

Otherwise yes, if I go deeper into source directories, I do see the Haskell logo pop up (but no GHC version).

Reply:
yea, I agree with it!

Reply:
This PR has kinda been ignored / neglected. If there is demand for it I can rewrite to fix it, but, I'm not sure right now.

Reply:
Oh certainly I was looking forward to it. That's not a demand, of course, we've all got lives.

Reply:
The main thing I was interested in was the detection and display of the `resolver` field in a `stack.yaml`, if such a file is present in the project.

Reply:
Alright